docs/resources/modelarts_resource_pool.md

```diff
@@ -24,62 +26,205 @@ resource "huaweicloud_modelarts_resource_pool" "test" {
+
+* `vpc_id` - (Optional, String, ForceNew) Specifies the VPC ID.
+
+  Changing this parameter will create a new resource.
+
+-> **NOTE:** Exactly one of `network_id`, `vpc_id` should be specified.
+
+* `subnet_id` - (Optional, String, ForceNew) Specifies the network ID of a subnet. It is mandatory when `vpc_id` is
+  specified.
+  
+  Changing this parameter will create a new resource.
+
+* `password` - (Optional, String, ForceNew) Specifies the password of the login user. It is mandatory when `vpc_id` is
+  specified and can not be specified if `network_id` is specified.
+
+  Changing this parameter will create a new resource.
+
+* `clusters` - (Optional, List, ForceNew) Specifies the list of the CCE clusters. It is mandatory when `vpc_id` is
+  specified and can not be specified if `network_id` is specified.
+  The [clusters](#ModelartsResourcePool_Clusters) structure is documented below.
+
+  Changing this parameter will create a new resource.
+
+* `workspace_id` - (Optional, String, ForceNew) Specifies the workspace ID, which defaults to **0**.
 
   Changing this parameter will create a new resource.
 
-* `workspace_id` - (Optional, String, ForceNew) Workspace ID, which defaults to 0.  
+* `description` - (Optional, String) Specifies the description of the dedicated resource pool.  
 
+* `charging_mode` - (Optional, String, ForceNew) Specifies the charging mode of the resource pool.
+  Valid values are **prePaid** and **postPaid**, defaults to **postPaid**.
   Changing this parameter will create a new resource.
 
-* `description` - (Optional, String) The description of the dedicated resource pool.  
+* `period_unit` - (Optional, String, ForceNew) Specifies the charging period unit of the resource pool.
+  Valid values are **month** and **year**. This parameter is mandatory if `charging_mode` is set to **prePaid**.
+  Changing this parameter will create a new resource.
+
+* `period` - (Optional, Int, ForceNew) Specifies the charging period of the resource pool.
+  If `period_unit` is set to **month**, the value ranges from **1** to **9**.
+  If `period_unit` is set to **year**, the value ranges from **1** to **3**.
+  This parameter is mandatory if `charging_mode` is set to **prePaid**.
+  Changing this parameter will create a new resource.
+
+* `auto_renew` - (Optional, String) Specifies whether auto-renew is enabled. Valid values are **true** and **false**.
+
+-> **NOTE:** `charging_mode`, `period_unit`, `period`, `auto_renew` must be specified when `vpc_id` is specified.
 
 <a name="ModelartsResourcePool_ResourceFlavor"></a>
 The `resources` block supports:
 
-* `flavor_id` - (Required, String) The resource flavor ID.  
+* `flavor_id` - (Required, String) Specifies the resource flavor ID.  
+
+* `count` - (Required, Int) Specifies the number of resources of the corresponding flavors.
+
+* `node_pool` - (Optional, String) Specifies the name of resource pool nodes. It can contain **1** to **50**
```

Comment:
can we writing like this:
```suggestion
* `node_pool` - (Optional, String) Specifies the name of resource node pool. It can contain **1** to **50**
```

Reply:
in reality, it is used to describe the name of the resource pool node, not a pool, so it should not be changed like this.